config/libc/mingw-w64.in

```diff
@@ -17,6 +17,16 @@ config MINGW_SECURE_API
     bool "Expose secure API prototypes"
     default y
 
+config MINGW_DEFAULT_MSVCRT
+    string "Default C Runtime to use"
+    default "msvcrt"
+    help
+      See mingw-w64 help for full list of possible values.
+      Most commonly one would want to upgrade this to ucrt
+      for a more modern C runtime.
+      Targeting specific older/newer MSVC versions is possible
+      as well though.
+
```

Comment:
I thought about that, but ultimately my thinking was: This option is so niche that anyone who'll want to mess with it will already know what they want to do. So I opted for the more flexible and slightly more future-proof approach (The list would need updated with every new MSVC version, even though almost nobody will ever want to actually use that.